public/static/app/perf/perf.test.js

```diff
@@ -98,4 +99,232 @@ describe("PerfPluginTests", function () {
 
         expect(converter(testdata)).toEqual(convertedData);
     })
+
+    it("perf results should be merged correctly", function() {
+        var merge = $filter("mergePerfResults");
+        var firstSample = {
+            "data": {
+              "results": [
+                {
+                  "end": 1550873331.840443,
+                  "name": "testone",
+                  "results": {
+                    "32": {
+                      "ops_per_sec": 8,
+                      "ops_per_sec_values": [
+                        8
+                      ]
+                    }
+                  },
+                  "start": 1550873204.842642,
+                },
+                {
+                  "end": 1550873784.308943,
+                  "name": "testtwo",
+                  "results": {
+                    "32": {
+                      "95th_read_latency_us": 8,
+                      "95th_read_latency_us_values": [
+                        8
+                      ],
+                      "99th_read_latency_us": 8,
+                      "99th_read_latency_us_values": [
+                        8
+                      ],
+                      "average_read_latency_us": 8,
+                      "average_read_latency_us_values": [
+                        8
+                      ],
+                      "ops_per_sec": 8,
+                      "ops_per_sec_values": [
+                        8
+                      ]
+                    }
+                  },
+                  "start": 1550873498.82311,
+                },
+                {
```

Comment:
First data != second data for ycsb_95read5update_secondary_reads.

Otherwise you can't tell if this actually worked correctly.

Reply:
This is intentional to test that values from the first sample will overwrite those in the second if there is a duplicate test name

Reply:
That is exactly my point, but if the values for first and second are exactly the same then how can you tell if it was overwritten.

Reply:
They're different though. The first sample has all the values set to 8, whereas the second has completely different values

Reply:
whoops I read them wrong